clang/lib/Gold/GoldParser.cpp

```diff
@@ -262,7 +555,9 @@ void Parser::parseArray(ArraySemantic S, llvm::SmallVectorImpl<Syntax *> &Vec) {
   appendTerm(Vec, List);
 
   while (true) {
-    // Obviously, stop at the end of the file.
```

Comment:
why add a bunch of whitespace here?

Reply:
I was doing something and removed it then forgot to fix thing. I have this fixed but not pushed yet.